include/abb_librws/rws_interface.h

```diff
@@ -351,6 +351,21 @@ class RWSInterface
    * \return bool indicating if the communication was successful or not. Note: NOT if it was parsed correctly.
    */
   bool getMechanicalUnitRobTarget(const std::string mechunit, RobTarget* p_robtarget);
+  
+  /**
+   * \brief A method for retriving the data of an RAPID symbol (parsed into a struct representing the RAPID data).
+   *
+   * \param task for the name of the RAPID task containing the RAPID symbol.
```

Comment:
It is supposed to be read "The parameter <name> is for the <description>", but yes it might be a bit unclear. Can maybe be changed in a "cleaning up" PR, since there are many places like this.